packages/rocketchat-action-links/server/actionLinkHandler.js

```diff
@@ -0,0 +1,58 @@
+//Action Links Handler. This method will be called off the client.
+
+Meteor.methods({
+	actionLinkHandler(fuctIndex, msgObj, userIdCalled) {
+		var rcDataBaseObj = RocketChat.models.Messages.findOne({ _id: msgObj._id });
+		// console.log(rcDataBaseObj);
+		
+		if (!msgObj) {
+			console.log("returned not msgObj");
+			return;
+		}
+		if (!userIdCalled) {
+			console.log("returned not called user");
+			return;
+		}
+		if (!rcDataBaseObj) {
+			console.log("returned not rcDataBaseObj");
+			return;
+		}
+		if (!rcDataBaseObj.actionLinks) {
+			console.log("returned not actionLinks");
+			return;
+		}
+
+		var roomCalled = RocketChat.models.Rooms.findOne({_id: rcDataBaseObj.rid});
+		// console.log("room: ", roomCalled);
+
+		//Get user who called method
+		var userCalled = RocketChat.models.Users.findOneById(userIdCalled, 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2673573741/issues/source?bid=3199634&fileBranchId=3333125#l29)